### PR TITLE
Added Unreal information to MR performance article

### PR DESCRIPTION
**Pull request by luis-valverde-ms**
Just mirrored what we already do for Unity.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@luis-valverde-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change. @hferrone

**Comment by luis-valverde-ms** [comment]
@sajidfarooq , I just added links to a few UE docs but there is more info in the article that is specific to Unity for which we don't have a UE equivalent. Would you mind giving the document a pass and add anything else you think is worth adding? We can do it via this PR or separately, whatever suits.

**Comment by hferrone** [comment]
Hi @luis-valverde-ms - all internal PRs need to go through the private repo in the future. Thanks for your contribution!

File: mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/understanding-performance-for-mixed-reality.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Below are some common tools to help you gather deep profiling information for yo
 - [Visual Studio Graphics Debuggers](https://docs.microsoft.com/visualstudio/debugger/graphics/visual-studio-graphics-diagnostics)
 - [Unity Profiler](https://docs.unity3d.com/Manual/Profiler.html)
 - [Unity Frame Debugger](https://docs.unity3d.com/Manual/FrameDebugger.html)
+- [Unreal Insights](../unreal/unreal-insights.md)
```
Comment (sajidfarooq) [inline mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/understanding-performance-for-mixed-reality.md:50]: I would add "Pix" here as well. It is especially useful for HL2 debugging

File: mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/understanding-performance-for-mixed-reality.md [hunk 2]
```diff
@@ -136,6 +137,8 @@ Object pooling is a popular technique to reduce the cost of continuous allocatio
 ## See also
 - [Performance recommendations for Unity](../unity/performance-recommendations-for-unity.md)
 - [Recommended settings for Unity](../unity/recommended-settings-for-unity.md)
+- [Performance recommendations for Unreal](../unreal/performance-recommendations-for-unreal.md)
+- [Material recommendations in Unreal](../unreal/unreal-materials.md)
 - [Optimize 3D models](https://docs.microsoft.com/dynamics365/mixed-reality/import-tool/optimize-models#performance-targets)
 - [Best practices for converting and optimizing real-time 3D Models](https://docs.microsoft.com/dynamics365/mixed-reality/import-tool/best-practices)
```
Comment (sajidfarooq) [inline mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/understanding-performance-for-mixed-reality.md:144]: Performance guidelines for Artists and Designers: https://docs.unrealengine.com/en-US/TestingAndOptimization/PerformanceAndProfiling/Guidelines/index.html

Though for VR, still very useful info: https://docs.unrealengine.com/en-US/SharingAndReleasing/XRDevelopment/VR/DevelopVR/ContentSetup/index.html

**Review by sajidfarooq — CHANGES_REQUESTED** [review]
Added some more useful links

**Comment by PRMerger20** [comment]
@luis-valverde-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change. @hferrone

**Comment by PRMerger13** [comment]
@hferrone : Thanks for your contribution! The author(s) have been notified to review your proposed change. @hferrone

**Pull request merged**